### ISSUE TITLE
Brain Midline Distortion

### ISSUE THREAD

**Issue by ivalenzuela84** [obophenotype/human-phenotype-ontology#9928]
**Brain Midline Distortion:**

**Synonyms** Midline shift

**Definition (free text, please give PubMed ID)** Deformity in midline structure of developing fetal brain ((DOI: 10.1002/uog.26140))

**Parent term (use hpo.jax.org/app)**
Abnormality of brain morphology HP:0012443

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Morphological findings on neurosonography and magneticresonance imaging (MRI) in fetuses with mild tubulinopathy. In brain MRI 90% of the fetus of a series with mild tubulinopathy present this finding of midline distortion (DOI: 10.1002/uog.26140). 
- Genes related to this phenotype TUBB, TUBB3, TUBA1A, TUBB2B 

**Your nano-attribution (ORCID)** https://orcid.org/0000-0003-2350-6058

**Comment by pnrobinson** [comment]
@ivalenzuela84 sorry for the very late reply -- I was relocating back to Germany last year and got way behind.
Do you know if this feature is also observed postnatally?
Also, it appears that
Distortion of anterior part of interhemispheric fissure
would be the more specific way of referring to this phenotype?

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000791
- label: Distortion of the anterior part of the interhemispheric fissure
- parents: [Abnormal subarachnoid space morphology (HP:0012703)](https://hpo.jax.org/app/browse/term/HP:0012703)
- pmid: PMID:34984691|PMID:23640781
- def: Change in the normal shape (displacement, asymmetry) of the anterior part of the interhemispheric fissure. This finding can be identified prenatally by sonography or magnetic resonance imaging.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9928
- synonyms:
